server/handlers.go

```diff
@@ -40,25 +36,6 @@ const (
 
 type HandlerPatternMap map[string]http.HandlerFunc
 
-type QROptions struct {
-	URL                  string `json:"url"`
-	ErrorCorrectionLevel string `json:"errorCorrectionLevel"`
-	Capacity             string `json:"capacity"`
-	AllowProfileImage    bool   `json:"withLogo"`
-}
```

Comment:
Ah, I see that something close to what I mention in this comment https://github.com/status-im/status-go/pull/3172/files#r1106508086 used to exist.